icons/qbittorrent.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>qbittorrent</title><path d="M21.624 4.7935c-2.1236-3.0817-5.806-4.8682-9.8823-4.785-4.0645.0857-7.6801 2.0293-9.636 5.1462-2.1442 2.9003-2.689 6.846-1.4585 10.555 1.2326 3.7146 4.0332 6.555 7.4323 7.5793 1.2263.473 2.5223.7052 3.8315.7052 2.4198 0 4.8834-.7936 7.0304-2.3284 3.3018-2.3596 5.1892-5.9944 5.0516-9.7278-.004-2.5668-.8515-5.1129-2.369-7.1445zm-3.656 15.5104c-2.8483 2.036-6.3205 2.5668-9.345 1.4026-2.9379-.8874-5.3253-3.326-6.3867-6.5238-1.0594-3.193-.605-6.5695 1.2514-9.0851 1.6908-2.6907 4.7896-4.3413 8.2896-4.4151.0817-.002.1626-.0028.244-.0028 3.4018 0 6.4635 1.5087 8.2422 4.0895 1.3252 1.7749 2.0547 3.971 2.0547 6.1849l.0008.032c.122 3.174-1.5042 6.2834-4.351 8.3178zm1.6173-11.874a3.0136 3.0136 0 0 0-1.1615-.9773c-.468-.2343-.9655-.3519-1.492-.3519-.9519 0-1.7342.419-2.347 1.2587V3.7547h-1.452v12.9082H14.48v-1.1708c.5752.9213 1.3679 1.382 2.379 1.382 1.016 0 1.8872-.4375 2.6135-1.3116.7256-.8749 1.0888-2.1136 1.0888-3.7163 0-.675-.0864-1.309-.2584-1.9017s-.4112-1.098-.7176-1.5147zm-1.2015 6.2433c-.4624.5982-1.0112.898-1.6455.898-.78 0-1.3984-.4077-1.8551-1.2235-.28-.4998-.4192-1.31-.4192-2.4311l-.0008.0008c0-1.1396.2312-2.0073.6935-2.6071.4632-.6046 1.0112-.9069 1.6455-.9069.6296 0 1.1648.2911 1.6056.8717.4455.5814.6695 1.4827.6695 2.703 0 1.1916-.2312 2.0905-.6935 2.6951zM9.5515 8.5618c-.608-.974-1.4416-1.461-2.5-1.461a3.26 3.26 0 0 0-1.8718.5805c-.5648.3822-.9976.9453-1.2984 1.6906-.296.7461-.444 1.5882-.444 2.527 0 1.5443.3656 2.7591 1.0968 3.646.7368.886 1.6135 1.33 2.6295 1.33a2.573 2.573 0 0 0 1.2903-.344c.4088-.2294.7263-.5166.952-.8628v4.5784h1.4519V7.3118H9.5515v1.25zm-.6696 6.1554c-.4464.5694-.9784.854-1.5967.854-.6464 0-1.1992-.299-1.6623-.8988-.4616-.6038-.6936-1.5083-.6936-2.711v-.0008c0-1.238.2183-2.148.6535-2.7295.4408-.587.976-.8805 1.6056-.8805.6344 0 1.1855.3143 1.6535.9421.4727.6286.7096 1.5643.7096 2.8094 0 1.1676-.2233 2.0393-.6696 2.6151z"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>qbittorrent</title><path d="M9.4058 20.2465V15.668c-.2255.3463-.5431.6334-.952.8629a2.5733 2.5733 0 0 1-1.2902.3438c-1.016 0-1.8927-.4438-2.6295-1.3299-.7312-.8869-1.0967-2.1016-1.0967-3.6459 0-.9389.148-1.781.444-2.527.3007-.7454.7335-1.3084 1.2983-1.6907a3.2597 3.2597 0 0 1 1.8719-.5806c1.0583 0 1.8919.487 2.4998 1.4611v-1.25h1.3064v12.9347h-1.452zm-4.4765-8.2851c0 1.2028.232 2.1072.6935 2.711.4632.5998 1.016.899 1.6624.899.6183 0 1.1503-.2848 1.5967-.8542.4463-.5758.6695-1.4475.6695-2.615 0-1.2452-.2368-2.1809-.7095-2.8095-.468-.6277-1.0192-.942-1.6535-.942-.6296 0-1.1648.2935-1.6056.8805-.4351.5814-.6535 1.4914-.6535 2.7294m9.5499 4.7023h-1.3464V3.7547h1.452v4.6048c.6127-.8397 1.3951-1.2588 2.347-1.2588.5264 0 1.024.1176 1.492.3519a3.014 3.014 0 0 1 1.1615.9772c.3064.4167.5456.9221.7176 1.5147.172.5926.2584 1.2268.2584 1.9017 0 1.6027-.3632 2.8414-1.0888 3.7163-.7263.8741-1.5975 1.3115-2.6134 1.3115-1.0112 0-1.804-.4606-2.3791-1.3819v1.1708m-.016-4.7463c0 1.1212.1392 1.9313.4192 2.4311.4567.8158 1.0751 1.2236 1.855 1.2236.6344 0 1.1832-.2999 1.6456-.898.4623-.6047.6935-1.5035.6935-2.6951 0-1.2204-.224-2.1217-.6695-2.703-.4408-.5807-.976-.8717-1.6055-.8717-.6344 0-1.1824.3022-1.6456.9068-.4623.5998-.6935 1.4675-.6935 2.6071M12 24c-1.6198 0-3.1914-.3173-4.671-.9432-1.429-.6044-2.7124-1.4696-3.8143-2.5715C2.4127 19.3833 1.5476 18.1.9432 16.671.3173 15.1914 0 13.6198 0 12c0-1.6198.3173-3.1914.9432-4.671.6044-1.429 1.4696-2.7123 2.5715-3.8143C4.6167 2.4127 5.9 1.5476 7.329.9432 8.8086.3173 10.3802 0 12 0c1.6198 0 3.1914.3173 4.671.9432 1.429.6044 2.7124 1.4696 3.8143 2.5715 1.102 1.102 1.9671 2.3852 2.5715 3.8142C23.6827 8.8086 24 10.3802 24 12c0 1.6198-.3173 3.1914-.9432 4.671-.6044 1.429-1.4696 2.7124-2.5715 3.8143-1.102 1.102-2.3852 1.9671-3.8143 2.5715C15.1914 23.6827 13.6198 24 12 24zM12 .75C5.7967.75.75 5.7967.75 12S5.7967 23.25 12 23.25 23.25 18.2033 23.25 12 18.2033.75 12 .75z"/></svg>
```

Reply:
@PratyushJayachandran - expanding the stroke in the source icon gives the red icon shown below. The circle around your proposed change is too thin. Could you please revise?
![image](https://github.com/simple-icons/simple-icons/assets/1830867/7dca0bb4-e2dc-4d89-bb32-2e7f8046780d)

Reply:
@adamrusted I updated my suggested change above as you requested.